workflows/Discord_Submission_Notify.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       uses: actions/checkout@master
     - name: Discord notification (1/2)
       env:
-        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK_SUBMISSION }}
```

Comment:
Also forgot this on line 26 in this file, but I can't comment on that line.

Reply:
Thanks got them all fix now